sale_report_salesperson_from_partner/report/sale_report.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    commercial_user_id = fields.Many2one(
```

Comment:
```suggestion
    user_id_from_partner = fields.Many2one(
```

I discarded `partner_user_id`, as it can be interpreted as "the partner of the user".

Reply:
How about user_from_partner_id?

Reply:
Yes, better to keep conventions, thanks.